### PR TITLE
add config for Faster RCNN

### PR DESCRIPTION
**Pull request by jerrywgz**

### CODE REVIEW

File: fluid/faster_rcnn/reader.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
 
 from roidbs import JsonDataset
 import data_utils
+from config import *
 
 
 class Settings(object):
```
Comment (qingqing01) [inline fluid/faster_rcnn/reader.py]: Can this `Settins`  be removed?

File: fluid/faster_rcnn/config.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    rpn_post_nms_top_n = 1000
+    # min size in RPN proposals
+    rpn_min_size = 0.0
+    # max number of detections
+    detectiions_per_im = 100
+    # NMS threshold used on RPN proposals
+    rpn_nms_thresh = 0.7
+
+
+class ModelConfig(object):
```
Comment (qingqing01) [inline fluid/faster_rcnn/config.py]: So many config, a little confused.

**Review by xingyuanbu — COMMENTED** [review]
Using splited class for different config seems unnecessary

File: fluid/faster_rcnn/train.py [hunk 3]
```diff
@@ -58,18 +59,20 @@ def train(cfg):
     rpn_reg_loss.persistable = True
     loss = loss_cls + loss_bbox + rpn_cls_loss + rpn_reg_loss
 
-    boundaries = [120000, 160000]
-    values = [learning_rate, learning_rate * 0.1, learning_rate * 0.01]
+    boundaries = cfg.lr_steps
+    gamma = cfg.lr_gamma
+    values = [learning_rate, learning_rate * gamma, \
+      learning_rate * gamma * gamma]
```
Comment (qingqing01) [inline fluid/faster_rcnn/train.py]: 如果 lr_steps数增多，values就不是3个，也会相应增多。这里需求自适应下。

Comment (jerrywgz) [reply fluid/faster_rcnn/train.py]: Done

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**